src/main/kotlin/tanvd/grazi/ide/language/props/PropsSupport.kt

```diff
@@ -0,0 +1,22 @@
+package tanvd.grazi.ide.language.props
+
+import com.intellij.lang.properties.psi.impl.PropertyValueImpl
+import com.intellij.psi.PsiElement
+import tanvd.grazi.grammar.GrammarChecker
+import tanvd.grazi.grammar.Typo
+import tanvd.grazi.ide.language.LanguageSupport
+
+class PropsSupport : LanguageSupport() {
```

Comment:
I would better rename it to `Properties`. At long last this is the name of plugin itself

Reply:
Done